build_tools/cmake/iree_macros.cmake

```diff
@@ -72,6 +88,28 @@ function(iree_package_dir PACKAGE_DIR)
   set(${PACKAGE_DIR} ${_PACKAGE_DIR} PARENT_SCOPE)
 endfunction()
 
+# iree_get_executable_path
+#
+# Gets the path to an executable in a cross-compilation-aware way. This is
+# meant to be used for binaries that are used to generate source files and
+# can only be built on host.
```

Comment:
Nit: it could be used on binaries that we expect to be built on host even if they technically could be built on device. They also don't necessarily have to be generating source files (iree_bytecode_module generates binary files). Basically anytime you're using a binary in a build step, you should use this, even if that binary *could* be built on the target and even if your output isn't a source file. Even if iree-translate started working on android, we'd still want iree_bytecode_module to use this, for example.


```suggestion
# Gets the path to an executable in a cross-compilation-aware way. This
# should be used when accessing binaries that are used as part of the build,
# such as for generating files used for later build steps.
```

Reply:
SG. Thanks for the suggestion!